.betterer.results

```diff
@@ -10336,13 +10336,13 @@ exports[`better eslint`] = {
       [216, 21, 3, "Unexpected any. Specify a different type.", "193409811"],
       [240, 21, 3, "Unexpected any. Specify a different type.", "193409811"]
     ],
-    "public/app/plugins/datasource/prometheus/metric_find_query.ts:3246701167": [
+    "public/app/plugins/datasource/prometheus/metric_find_query.ts:1417569751": [
       [62, 70, 3, "Unexpected any. Specify a different type.", "193409811"],
       [83, 72, 3, "Unexpected any. Specify a different type.", "193409811"],
       [96, 72, 3, "Unexpected any. Specify a different type.", "193409811"],
       [122, 70, 3, "Unexpected any. Specify a different type.", "193409811"],
       [140, 43, 35, "Do not use any type assertions.", "2712117061"],
-      [175, 70, 3, "Unexpected any. Specify a different type.", "193409811"]
+      [188, 70, 3, "Unexpected any. Specify a different type.", "193409811"]
```

Comment:
Hmm feels weird that this small change made such a difference. I thought you had a problem in some test code.